src/Uno.UI/UI/LayoutHelper.cs

```diff
@@ -10,10 +12,13 @@ namespace Uno.UI
 {
 	internal static class LayoutHelper
 	{
+		[Pure]
```

Comment:
I'm curious, can I have more info about why are we adding [Pure] ?

Reply:
For various reasons.  Since it's a pure method (<https://en.wikipedia.org/wiki/Pure_function>), it can help the build tooling to better optimise the method : if for any reason the returned value is not used, the call to this method could be removed.

Also, using tools like Resharper, you'll get a notification if you're not using the return value.

Reply:
Thanks for the details @carldebilly !